.github/workflows/sphinx-tuttest.yml

```diff
@@ -0,0 +1,67 @@
+name: doc-test
+
+on: [push, pull_request]
+
+jobs:
+  test-sphinx-docs:
+    runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
```

Comment:
Can you log an issue to auto-generate this output? (Fixing not blocking merge)

Reply:
I opened the issue: https://github.com/SymbiFlow/symbiflow-examples/issues/94